test/tests/neutronics/openmc_xml/tests

```diff
@@ -41,6 +41,18 @@
                   "(with 60 batches) to ensure correctness."
     capabilities = 'openmc'
   []
+  [check_statepoint_exists]
+    type = CheckFiles
+    input = openmc.i
+    check_files = 'statepoint.50.h5'
+    cli_args = 'Problem/batches=50'
```

Comment:
Can we choose a different number? In this directory are some inputs which will already be creating a file named `statepoint.50.h5`, so based on the test execution order it's theoretically possible that one of the other tests makes a 50-batch statepoint but we fail to, although the test would still pass. The best fix would be to put your test in its own directory, but I think using a batch number unique to this input (within this directory) will also do the trick.

Reply:
That's a good point. Is the execution order always first to last? I could do this test first. Otherwise, I need to update the XML to match whatever is in the `cli_args`. I think that is fine too as long as it isn't 60 for the `batches` test or 8 for the `skip_statepoint` test . I'm happy to put this in a diff directory too